src/Compilers/Core/Portable/SymbolDisplay/SymbolDisplayMiscellaneousOptions.cs

```diff
@@ -74,5 +74,10 @@ public enum SymbolDisplayMiscellaneousOptions
         /// Append '!' to non-nullable reference types.
         /// </summary>
         IncludeNotNullableReferenceTypeModifier = 1 << 8,
+
+        /// <summary>
+        /// Explicitly put `managed` as the calling convention for function pointer types.
```

Comment:
>Explicitly put [](start = 12, length = 14)

Minor: Perhaps just "Use `managed` as ..."

Reply:
Well, that's not quite accurate either, because it's just that we explicitly add `managed` as syntax, but only for `managed` function pointers.

---
In reply to: [467314142](https://github.com/dotnet/roslyn/pull/46467#discussion_r467314142) [](ancestors = 467314142)

Reply:
Rewording this to:

> Use `managed` keyword for managed function pointer types.

---
In reply to: [467318739](https://github.com/dotnet/roslyn/pull/46467#discussion_r467318739) [](ancestors = 467318739,467314142)